pkg/specgen/generate/kube/kube.go

```diff
@@ -260,6 +262,10 @@ func ToSpecGen(ctx context.Context, opts *CtrSpecGenOptions) (*specgen.SpecGener
 		}
 	}
 
+	if !useLogOpts && rtc.Containers.LogPath != "" {
+		s.LogConfiguration.Path = rtc.Containers.LogPath
+	}
+
```

Comment:
you do not need the useLogOpts bool, just move this condition above the logOptions loop.
If the user sets it on the cli it still gets set inside the loop correctly and thus overrides the default.